src/gulpfile.js

```diff
@@ -160,7 +160,10 @@ gulp.task('common:build', ['clean'], function() {
     return gulp.start('copy:vendor-css', 'copy:vendor-images', 'sass', 'collectstatic');
 });
 
-gulp.task('test', sequence(['jshint',
+gulp.task('test', sequence([
+            'development',
+            'production',
```

Comment:
I'm not sure what issues are occurring on Jenkins, just pointing out that running development here isn't going to do anything because production is just going to delete everything that got copied over during the development task.